internal/cli/alerts/settings_fields_type.go

```diff
@@ -32,13 +32,16 @@ func (opts *ConfigFieldsTypeOpts) init() error {
 	return err
 }
 
+var matcherFieldsTemplate = `{{range .}} {{.}} {{end}}
```

Comment:
[q] does this print extra empty spaces before and after the value? if so we should remove them
```suggestion
var matcherFieldsTemplate = `{{range .}}{{.}}{{end}}
```

Reply:
with
```
 TYPE_NAME  HOSTNAME  PORT  HOSTNAME_AND_PORT  REPLICA_SET_NAME  SHARD_NAME  CLUSTER_NAME
```

without
```
TYPE_NAMEHOSTNAMEPORTHOSTNAME_AND_PORTREPLICA_SET_NAMESHARD_NAMECLUSTER_NAME
```

Reply:
ok, that makes sense, then lets make this 
```go
var matcherFieldsTemplate = "{{range .}}{{.}}\n{{end}}"

Reply:
this is the output

```
TYPE_NAME
HOSTNAME
PORT
HOSTNAME_AND_PORT
REPLICA_SET_NAME
SHARD_NAME
CLUSTER_NAME
```